server/utils.go

```diff
@@ -103,6 +103,20 @@ func decrypt(key []byte, text string) (string, error) {
 	return string(unpadMsg), nil
 }
 
+func parseOwner(full, baseURL string) (string, string) {
+	if baseURL == "" {
+		baseURL = "https://github.com/"
+	}
+	full = strings.TrimSuffix(strings.TrimSpace(strings.Replace(full, baseURL, "", 1)), "/")
```

Comment:
0/5 use `url.Parse` and `Path.Split` - more idiomatic?

Reply:
@levb this was how it was already being used in parseOwnerAndRepo so just did the same.  Using the url package probably makes the most sense in this case though